src/RoadPreview/RoadPreview.tsx

```diff
@@ -0,0 +1,263 @@
+import {
+  Box,
+  Divider,
+  Link,
+  Stack,
+  Tooltip,
+  Typography,
+  useTheme
+} from "@mui/material";
+import { LabelChipGroup, LabelChipGroupProps } from "../LabelSelector";
+
+import Asam from "../../static/asam.png";
+import AttachFileIcon from "@mui/icons-material/AttachFile";
+import CarMaker from "../../static/carMaker.svg";
+import DateRangeOutlinedIcon from "@mui/icons-material/DateRangeOutlined";
+import NoWrapTypography from "../NoWrapTypography/NoWrapTypography";
+import NumbersIcon from "@mui/icons-material/Numbers";
+import React from "react";
+import { RoadPreviewProps } from "./RoadPreview.types";
+import TruncatedTooltip from "../TruncatedTooltip";
+import UserAvatar from "../UserAvatar/UserAvatar";
+import VersionChip from "../VersionChip/VersionChip";
+
+/**
+ * RoadPreview component for visualizing Road information about specific road
+ */
+
+export default function RoadPreview({
```

Comment:
we should change to using named exports as per the new style guide that we use in VIRTO.
https://github.com/IPG-Automotive-UK/virto/blob/qa/STYLEGUIDE.md#exports

```suggestion
export function RoadPreview({
```